app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -442,6 +442,11 @@ private void play(long position, @Nullable Integer forcedSubtitleIndex) {
             position = 0;
         }
 
+        if (mFragment == null) {
+            Timber.i("mFragment is null, returning");
```

Comment:
Maybe this should be a warning?

Reply:
Good call, I've also updated the earlier log statement from debug to info as it is quite useful.